assemblyline/common/net.py

```diff
@@ -32,6 +33,9 @@ def is_valid_domain(domain: str) -> bool:
 
 
 def is_valid_ip(ip: str) -> bool:
+    if not isinstance(ip, str):
+        return False
```

Comment:
I think it is better to fail noisily than to silently return false when the caller has a type error, to avoid hiding bugs